example/package.json

```diff
@@ -20,9 +20,9 @@
     "build": "npm run clean && npm run exe32 && npm run deb32 && npm run exe64 && npm run deb64"
   },
   "devDependencies": {
-    "electron-installer-debian": "^0.6.0",
-    "electron-packager": "^9.0.0",
     "electron": "~1.7.0",
+    "electron-installer-debian": "file:..",
```

Comment:
Is this intentional?

Reply:
Yes, I thought it would be better for the example to run the latest version rather than an old specific one, and even better to just get it from the parent directory rather than online. I don't think it's a good idea to have to update the dependency here everytime we release a new version, especially if we're moving to `1.0.0` or any future version

Reply:
That makes sense, I just worry about the GitHub issues that will come when people copy/paste into their project and it doesn't work the way they expect.

Reply:
That also makes sense! I was thinking of the people using the example for testing and submitting issues but not as a reference to use in their projects. How about we leave it as `*`? If they copy/paste it will work, and if a new release breaks their app, they can probably handle that.

Reply:
I'll begrudgingly add the one asterisk back: https://github.com/unindented/electron-installer-debian/commit/d3af805f44947c6b3201a8a226275aa244579c92#diff-012ac553d93d987b8d34ad013052bc96

:smile:

Reply:
I was about to post a comment mentioning that commit. At least it's only one dependency, and it's the only one that makes sense to have it as an asterisk.
Anyway, it seems this PR focused on the asterisk rather than the umask.  :satisfied:
Regarding the umask, I like your idea, better to test for the umask and just warn the user, so they can see fit what to do about it. I'll push those changes over the weekend.